modules/azure/log_analytics_workbook/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "location" {
+  type        = string
+  description = "A datacenter location in Azure."
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "Name of the resource group."
+}
+
+variable "workbook_name" {
```

Comment:
You can make variables required if they are (all?)

Reply:
They should be required based what i found :
- All defined variables must have values in order to run Terraform code.
- Once you set a default value for a variable, it becomes optional.